From b841905bf54c2c07d7968a82f696cfd5b8c2a7e1 Mon Sep 17 00:00:00 2001
From: Han Xu <han.xu@nxp.com>
Date: Mon, 9 May 2016 17:36:01 -0500
Subject: [PATCH 1051/5242] MLK-12769: ARM: dts: fix the optional QSPI pin
 conflict with SD2

commit  643560031bed84675f8f74319e05b11565ab4d8a from
https://source.codeaurora.org/external/imx/linux-imx.git

The optional QSPI pin should be only enabled on reworked board, fix the
issue which conflict with SD2.

Signed-off-by: Han Xu <han.xu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts
index b0557a9..45a4e0e 100644
--- a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts
@@ -666,6 +666,7 @@
 				MX6UL_PAD_NAND_CE1_B__QSPI_A_DATA02   0x70a1
 				MX6UL_PAD_NAND_CLE__QSPI_A_DATA03     0x70a1
 				MX6UL_PAD_NAND_DQS__QSPI_A_SS0_B      0x70a1
+#ifdef REWORKED_ENABLE_ALL_QSPI
 				MX6UL_PAD_NAND_DATA07__QSPI_A_SS1_B   0x70a1
 				MX6UL_PAD_NAND_RE_B__QSPI_B_SCLK      0x70a1
 				MX6UL_PAD_NAND_DATA02__QSPI_B_DATA00  0x70a1
@@ -674,6 +675,7 @@
 				MX6UL_PAD_NAND_DATA05__QSPI_B_DATA03  0x70a1
 				MX6UL_PAD_NAND_WE_B__QSPI_B_SS0_B     0x70a1
 				MX6UL_PAD_NAND_DATA00__QSPI_B_SS1_B   0x70a1
+#endif
 			>;
 		};
 
-- 
1.7.9.5

